internal/tools/aws/s3.go

```diff
@@ -35,6 +35,22 @@ func (a *Client) s3EnsureBucketCreated(bucketName string, logger log.FieldLogger
 		return errors.Wrap(err, "unable to block public bucket access")
 	}
 
+	_, err = a.Service().s3.PutBucketEncryption(&s3.PutBucketEncryptionInput{
```

Comment:
we might need to add this permission in the provisioner user
```
s3:PutEncryptionConfiguration
```

Reply:
I will check if we need

for now lets /hold this and not merge

Reply:
Thank you for checking that!

Reply:
need to merge and update the envs first: https://github.com/mattermost/mattermost-cloud-monitoring/pull/161